src/psyclone/psyGen.py

```diff
@@ -1456,9 +1458,8 @@ class CodedKern(Kern):
     :type call: :py:class:`psyclone.parse.algorithm.KernelCall`.
     :param parent: the parent of this Node (kernel call) in the Schedule.
     :type parent: sub-class of :py:class:`psyclone.psyir.nodes.Node`.
-    :param bool check: Whether or not to check that the number of arguments \
-                       specified in the kernel meta-data matches the number \
-                       provided by the call in the Algorithm layer.
+    :param bool check: whether to check for consistency between the \
+        kernel metadata and the algorithm layer. Defaults to True.
 
     :raises GenerationError: if(check) and the number of arguments in the \
```

Comment:
This routine no longer raises this exception.

Reply:
Oops.